src/github.com/getlantern/flashlight/geolookup/geolookup.go

```diff
@@ -31,23 +31,24 @@ var (
 	service  *ui.Service
 	client   atomic.Value
 	cfgMutex sync.Mutex
-	location atomic.Value
+	country  atomic.Value
+	ip       atomic.Value
 )
 
-func GetLocation() *geolookup.City {
-	l := location.Load()
-	if l == nil {
-		return nil
+func GetIp() string {
+	c := ip.Load()
+	if c == nil {
```

Comment:
Can we avoid this check each time if we set the `ip` atomic value as "" from the beginning?